hathor/transaction/resources/mining.py

```diff
@@ -112,7 +113,11 @@ def render_POST(self, request):
         request.setHeader(b'content-type', b'application/json; charset=utf-8')
         set_cors(request, 'GET')
 
-        data = json_loadb(request.content.read())
+        try:
+            data = json_loadb(request.content.read())
+        except JSONDecodeError as exc:
+            raise APIError('Cannot decode JSON') from exc
```

Comment:
Should we add this error and its returned status_code to the API docs?

Reply:
I reverted these changes because the decorator `@api_catch_exceptions` will handle these exceptions. Even though I think we should rethink this decorator, we should do it in a separate PR.